docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>This make me thinking that we should have better default names assigned to built-in analysis functions... Should I create an issue for that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Sure that seems reasonable to me, and I would be curious about your thoughts on a scheme! I wasn't too fond of this one myself.</p>

Reply:
I created this one:
https://github.com/lincc-frameworks/tape/issues/300